mm2src/coins/tendermint/tendermint_coin.rs

```diff
@@ -1769,17 +1759,17 @@ impl TendermintCoin {
                 let events_string = format!("claim_htlc.id='{}'", htlc_id);
                 let request = GetTxsEventRequest {
                     events: vec![events_string],
-                    pagination: None,
                     order_by: TendermintResultOrder::Ascending as i32,
+                    // FIXME: Put correct data here
+                    ..Default::default()
```

Comment:
Are we going to handle this note in this PR?

Reply:
Yup, that's just a reminder for me to ask you about it :).

The new API now accepts a `page` and a `limit` instead of the deprecated `pagination` object:
```rust
    /// pagination defines a pagination for the request.
    /// Deprecated post v0.46.x: use page and limit instead.
    #[deprecated]
    #[prost(message, optional, tag = "2")]
    pub pagination: ::core::option::Option<super::super::base::query::v1beta1::PageRequest>,
    ...
    /// page is the page number to query, starts at 1. If not provided, will default to first page.
    #[prost(uint64, tag = "4")]
    pub page: u64,
    /// limit is the total number of results to be returned in the result page.
    /// If left empty it will default to a value to be set by each app.
    #[prost(uint64, tag = "5")]
    pub limit: u64,
```

I think `page: 0` & `limit: 0` should be equivalent to `pagination: None`, but I need to make sure this is the case. I am confused by the wording "if not provided/if left empty", this should be like "if set to zero" instead I guess.

Reply:
You can use page and limit 1 and we are looking for specific tx by giving the HTLC id.